docs/user/gen-docs/kyma_alpha_diagnose_logs.md

```diff
@@ -0,0 +1,32 @@
+# kyma alpha diagnose logs
+
+Aggregate error logs from pods belonging to enabled Kyma Modules.
+
+## Synopsis
+
+Some better long description
+
+```bash
+kyma alpha diagnose logs [flags]
+```
+
+## Flags
+
+```text
+  -f, --format string           Output format (possible values: json, yaml)
+      --lines int64             Max lines per container (default "200")
```

Comment:
We don't know the priority between `--lines` and `--since`.
Shouldn't these flags be exclusive?

Reply:
Marked as exclusive